go/test/endtoend/vtgate/queries/multi_query/multi_query_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2025 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package multi_query
+
+import (
+	"fmt"
+	"testing"
+
+	"vitess.io/vitess/go/test/endtoend/utils"
+)
+
+// TestMultiQuery tests the new way of handling queries in vtgate
+// that runs multiple queries together.
+func TestMultiQuery(t *testing.T) {
```

Comment:
may be just add grpc client here for the same test.

Reply:
Done now 👍